test/functional/signature/version_sig_check.bats

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -L -n test-bundle1 -f /file_1 "$TEST_NAME"
+       	create_version -p "$TEST_NAME" 20 10
+        update_bundle "$TEST_NAME" test-bundle1 --add file_3
+
+}
+
+@test "SIG019: Verify signature for latest_version" {
+	
+	# Verify the latest_version signature though openssl
+	run sudo sh -c "openssl smime -verify -in $WEBDIR/version/latest_version.sig -inform DER -content $WEBDIR/version/latest_version -purpose any -CAfile $FUNC_DIR/Swupd_Root.pem"
+	expected_output=$(cat <<-EOM
+				Verification successful
+				20
+			EOM
+			)
+
+	assert_is_output "$expected_output"
+
+	# Update succeeds as verification is successful
+	run sudo sh -c "$SWUPD update $SWUPD_OPTS"
```

Comment:
I don't see a lot of benefit from this test case, I don't think is really testing what is attempting to test. I would think it would be more useful to maybe remove (rename) the signed file and make sure the command fails as it should then add it back and see the command work successfully now, or something along the lines of that

Reply:
Thanks for catching this. I was testing the wrong outcome. I will change.

Reply:
This is corrected now. Now the right thing is tested.